jobserver/views/workspaces.py

```diff
@@ -180,6 +182,12 @@ def get_initial(self):
         # setting as a default for the form which the user can override.
         return {"will_notify": self.workspace.should_notify}
 
+    def get_repo_is_private(self):
+        owner = furl(self.workspace.repo).path.segments[0]
```

Comment:
Should this be a method on `workspace`? `get_repo_owner` or something like that? Or even a `repo_owner` property